tidb-cloud/create-tidb-cluster.md

```diff
@@ -72,11 +72,11 @@ To create a Developer Tier cluster, take the following steps:
 
 4. View the cluster size of the Developer Tier, and then click **Create**.
 
-    Your TiDB Cloud cluster will be created in approximately 5 to 15 minutes.
+   The cluster creation process starts and the **Security Quick Start** dialog box is displayed.
 
-5. During or after the cluster creation, click **Security Quick Start** in the left cluster pane.
+5. In the **Security Quick Start** dialog box, set the root password and allowed IP addresses to connect to your cluster, and then click **Apply**.
 
-6. In the **Security Quick Start** dialog box, set the root password to access your cluster, add the IP addresses to connect to your cluster, and then click **Apply**.
+    Your TiDB Cloud cluster will be created in approximately 5 to 15 minutes.
```

Comment:
```suggestion
    Your TiDB Cloud cluster will be created in 5 to 15 minutes.
```
5 to 15 is already an approximate value. “Approximately” seems unnecessary.

Reply:
it might exceed 15 minutes sometime😂